containers/deploy/candlepin/routes.yml

```diff
@@ -20,3 +20,14 @@
                   targetPort: port-8080-tcp
   tags:
     - start
+- name: Destroy route
+  openshift_v1_route:
+      state: absent
+      resource_definition:
+          apiVersion: v1
+          kind: Route
+          metadata:
+              name: foreman-8080
+              namespace: foreman
```

Comment:
Instead of using `resource_definition` here can you just do:

```yaml
openshift_v1_route:
  name: foreman-8080
  namespace: foreman
  state: absent
```

Reply:
Also this for all of the other things ...